src/root/topbar.tt

```diff
@@ -143,6 +143,10 @@
         <a class="dropdown-item" href="/github-redirect?after=[% c.req.path %]">Sign in with GitHub</a>
         <div class="dropdown-divider"></div>
       [% END %]
+      [% IF c.config.enable_oidc_login %]
+        <a class="dropdown-item" href="/oidc-redirect?after=[% c.req.path %]">Sign in with OIDC</a>
```

Comment:
Would also be nice to be able to change the name of the provider so it can be consistent in an organization.